Src/Localization/English/ButtonTutorial.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
 <html><head>
-<title>Custom Start Buttons</title>
```

Comment:
What is the purpose of these changes in `.html` files?
And how is it related to `Change window title Help -> Open-Shell Help`?

Reply:
**What is the purpose of these changes in .html files?**
When localizing, we change the same fields by replacing the words of one language with another.
If the field - <title> tag is located in the same place, on the same line, then this speeds up its search, editing and replacing words with another language. In this case, the <title> tag
in all files is on the 4th line. Unification and ease of editing.
![image](https://user-images.githubusercontent.com/25941601/215325022-54d18600-d8fe-4a47-ab84-2a8e89e2aa69.png)


**And what does this have to do with Change Help Window Title -> Open-Shell Help?**
Sorry, this has nothing to do with changing the title of the help window. My omission - not enough well mastered GitHub Desktop...

Reply:
Oki. I guess it makes sense to have these files as similar as possible (so the structure of files is the same, only localized strings will differ).

But please, do such change in separate commit. So it won't mix with other (_real_) changes and makes it easier to review.

Reply:
Thanks for the comments and clarifications.
I initially set goals simply to make it possible to fully localize Open-Shell.
And having edited the Strings resources in StartMenuDLL.dll and SKIN in Immersive.skin7/Immersive.skin, I realized that it is necessary to make changes in the sources.
It turned out in some way confusion + Visual Studio warning.
I'm confused myself...

Briefly speaking.
There is a common task - Full localization of Open-Shell, it consists of translating resource strings + html help files.
1) I made minimal changes to the html files (English) + added two chapters to help.
2) Added a title to the Help window in all languages
3) Added resource strings to StartMenu/StartMenuDLL/StartMenuDLL.rc
4) Changed (because VS warning!)
src/Skins/Immersive/Immersive.vcxproj
Src/Skins/Immersive/Immersive.vcxproj.filters

src/Skins/Immersive/Immersive7.vcxproj
Src/Skins/Immersive/Immersive7.vcxproj.filters

How to do better? To make it more convenient for everyone.
Should I delete my repository along with the commits and create a new one by cloning?
And then create commits on points 1-4

Or will you use p. 3, p. 4

I'll try not to create chaos in the future. ;-)

Reply:
> Should I delete my repository along with the commits and create a new one by cloning?

There is no need to delete anything.
You can modify commits on your branch via _git interactive rebase_ (just google for it if you are not familiar) and then _force push_.

The advantage of git is that you can do everything locally on your machine and then once commits look good you can just push them to remote repository.

> How to do better? To make it more convenient for everyone.

Simply keep in mind that it is better to not mix unrelated things in single commit.

> I made minimal changes to the html files (English) + added two chapters to help.

From this description you can see you did two independent things.
So one commit should contain just adjustments in html files, that basically doesn't add anything new, just allows easier comparison of those files across languages.
The other commit should add new content to the help.

> Changed (because VS warning!)

This is already fixed on master (see my other comment), so there is no need for this change anymore.

Reply:
Another advice.
If you are not familiar with _git_ and the way to work with commits locally, then I'd recommend to create one PR per change you want to do.

So if you want to change 4 different things then just create separate PR for each one (one PR after another).
That way it may be easier to focus on each partial change and commits in the PR will get squashed in the end (as they will be related to the same thing).

What about that?

Reply:
Everything is very clear - about one action - one commit.
Ща вкурим про `git interactive rebase` и всё. ;-)